05-production.md

```diff
@@ -0,0 +1,37 @@
+# Production Deployment.
+Now that we've provisioned a *development* version of the service, you will provision a *production* version. You'll repeat many of the previous steps you've just completed. When finished you'll have an entirely new *production* instance of *Wild Rydes* running along side your *development* instance.
```

Comment:
Reading this makes it sound like we'll be redoing everything we just did: "You'll repeat many of the previous steps you've just completed." When I read it just now, it sounded really daunting. Then I scrolled through the rest of the page and remembered it wasn't *that* many steps :).

We could drop the sentence and everything would still be fine. Or we could state specifically what we're going to do. Either would be fine with me :).

Reply:
I think module 5 isn't necessary because it really is just repeating a bunch of the steps from the previous modules. BUT, it shows off how easy it is to deploy a new instance and the lack of detailed directions I think help to assess how good and memorable the previous directions were. Observing people in a training setting and how well they do with module 5 is I think a way to make observations about the quality of modules 1-4 and the intuitiveness of the product as a whole.